code/game/gamemodes/cult/items/tome.dm

```diff
@@ -107,20 +188,29 @@
 			log_and_message_admins("created \an [chosen_rune] at \the [A.name] - [user.loc.x]-[user.loc.y]-[user.loc.z].") //only message if it's actually made
 
 			var/obj/effect/rune/R = new(get_turf(scribe), SScult.runes_by_name[chosen_rune])
-			to_chat(scribe, SPAN_CULT("You finish drawing the Geometer's markings."))
+			to_chat(scribe, SPAN_CULT("The chalk's blood spent into the rune, you finish drawing the Geometer's markings with a mental prayer."))
 			R.blood_DNA = list()
 			R.blood_DNA[scribe.dna.unique_enzymes] = scribe.dna.b_type
 			R.color = scribe.species.blood_color
 			R.filters = filter(type="drop_shadow", x = 1, y = 1, size = 4, color = scribe.species.blood_color)
+			playsound(scribe, pick('sound/bureaucracy/chalk1.ogg','sound/bureaucracy/chalk2.ogg'), 50, FALSE)
+			
 	else
-		to_chat(user, SPAN_CULT("The book seems full of illegible scribbles."))
-
-/obj/item/book/tome/examine(mob/user)
-	..(user)
-	if(!iscultist(user) || !isobserver(user))
-		to_chat(user, "An old, dusty tome with frayed edges and a sinister looking cover.")
+		to_chat(user, SPAN_CULT("The book seems full of illegible scribbles and bizarre symbols.   Some of the pages have[bloody] spiked metal studs poking out from them, in the general shape of a hand."))
+		if(inserted_item)
+			to_chat(user, SPAN_WARNING("A[bloody] piece of chalk lies nestled within a compartment in the back."))
+
+/obj/item/book/tome/examine(mob/user, var/distance = -1, var/infix = "", var/suffix = "", var/show_blood = FALSE)
+
+	. = ..()
+	
+	if(iscultist(user) || isobserver(user))
+		to_chat(user, "The unholy scriptures of Nar-Sie, The One Who Sees, The Geometer of Blood.  Contains the details of every ritual his followers could think of. Most of these are useless, though.\ 
+		</br>At the end of the listing for each ritual lies a[bloody] hand-shaped set of spikes embedded into the page, with which you imbue your blood into chalk for the ritual's magic.")
+		if(inserted_item)
+			to_chat(user, SPAN_NOTICE("A[bloody] piece of chalk lies nestled in its compartment."))
```

Comment:
```suggestion
			to_chat(user, SPAN_NOTICE("A [bloody] piece of chalk lies nestled in its compartment."))
```

Reply:
I did it that way, because if you look at all the spaces [bloody] is in, if I were to just correct it as you ask and remove the space from " bloody", you'd wind up with double spaces whenever there is no [bloody].

Reply:
I see